pos-print/src/test/java/com/clouway/pos/print/persistent/InMemoryDatastoreRule.kt

```diff
@@ -0,0 +1,26 @@
+package com.clouway.pos.print.persistent
+
+import com.github.fakemongo.Fongo
+import com.mongodb.client.MongoDatabase
+import org.junit.rules.ExternalResource
+
+/**
+ *@author Borislav Gadjev <borislav.gadjev@clouway.com>
+ */
+class InMemoryDatastoreRule : ExternalResource() {
```

Comment:
Ok, I though I already had DatastoreRule.kt, but it appears I dont't. DatastoreRule it is.